src/KOKKOS/sna_kokkos_impl.h

```diff
@@ -322,7 +337,7 @@ void SNAKokkos<DeviceType>::compute_ui(const typename Kokkos::TeamPolicy<DeviceT
 
   const double wj_local = wj(iatom, jnbor);
   const double rcut = rcutij(iatom, jnbor);
-  const int jpos = element(iatom, jnbor)*idxu_max;
+  const int jpos = element(iatom, jnbor);
```

Comment:
jpos should be changed to jelem for clarity now that it is not influenced by idxu_max. This will also make it consistent with the rest of the methods.

I tried to make a Pull Request to your branch, but it was not showing up in the drop down menu.

Reply:
That's odd---I can quickly make that change, though, just a moment.